.github/workflows/build-icu.yml

```diff
@@ -32,10 +32,10 @@ jobs:
           ref: ${{ github.event.inputs.cldr-ref }}
           lfs: false
           repository: ${{ github.event.inputs.cldr-repo }}
-      - name: Set up JDK 1.8
+      - name: Set up JDK 11
```

Comment:
does the name need the version number, or would "Set up JDK" do as well?

Reply:
done